spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb

```diff
@@ -420,14 +420,22 @@ def parse_entities(namespace_pod, namespace_config)
                               'required'    => true
                              }
                            ],
-                           :objects    => []
+                           :objects    => [],
+                           :labels     => {
```

Comment:
can you also add an example of regular labels in this test?

Reply:
done